src/meson.build

```diff
@@ -368,6 +381,8 @@ libharfbuzz = library('harfbuzz', hb_sources,
   soversion: hb_so_version,
   version: version,
   install: true,
+  #vs_module_defs: harfbuzz_def,
```

Comment:
If I enable `vs_module_defs` I will get https://ci.appveyor.com/project/harfbuzz/harfbuzz/builds/32769174/job/jrekl36yixwitpgo#L2131 (and lots warnings above)